index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Hello Microverse">
+    <meta name="keywords" content="HTML, CSS, Linter">
+    <meta name="author" content="Oussedit">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <header>
+            <a href="">
+                <div id="logo">My logo</div>
+            </a>
+            <nav>
+                <div><img src="./assets/ham.svg" alt="" > </div>
+            </nav>
+        </header>
+    </div>
+    <main>
+        <section id="sec1">
+
+            <div>
+                <h1 id="nm">I’m Mariane<br><br>
+                    Glad to see<br><br> you!</h1>
+
+                <p class="desc">I’m a software developer! I can help you build a product , feature or website Look
+                    through
+                    some of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+
+                <a href="#" class="lnk">LET’S CONNECT</a>
```

Comment:
- [ ] As the previous reviewer has suggested, Kindly ensure that your mobile view looks exactly like the Figma design, I can see that your texts are still overlapping each other. I think the issue is a result of the absolute positioning you used for the texts. Kindly fix it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/211898660-497eae22-4b3d-46ee-bda2-654e85d29c9e.png)